### DISCUSSION TITLE
Update 28 (9.0.1) to 28 (9.0.3) failed

### DISCUSSION THREAD

**Discussion by MK796** [nextcloud/all-in-one#5069] [Questions] [answered]
### Steps to reproduce
1. Try to update from to Update (28) 9.0.1 to (28) 9.0.3

### Expected behavior:
The update should run without any issue.

### Actual behavior:
Watchtower unable to kill AIO container.
Update Failed.

#### Other valuable info:
Log of the watchtower container:

time="2024-07-29T19:25:58Z" level=debug msg="Sleeping for a second to ensure the docker api client has been properly initialized."
time="2024-07-29T19:25:59Z" level=debug msg="Making sure everything is sane before starting"
time="2024-07-29T19:25:59Z" level=info msg="Watchtower 1.7.1"
time="2024-07-29T19:25:59Z" level=info msg="Using no notifications"
time="2024-07-29T19:25:59Z" level=info msg="Only checking containers which name matches \"nextcloud-aio-mastercontainer\""
time="2024-07-29T19:25:59Z" level=info msg="Running a one time update."
time="2024-07-29T19:25:59Z" level=debug msg="Checking containers for updated images"
time="2024-07-29T19:25:59Z" level=debug msg="Retrieving running containers"
time="2024-07-29T19:25:59Z" level=debug msg="Trying to load authentication credentials." container=/nextcloud-aio-mastercontainer image="nextcloud/all-in-one:latest"
time="2024-07-29T19:25:59Z" level=debug msg="No credentials for index.docker.io found" config_file=/config.json
time="2024-07-29T19:25:59Z" level=debug msg="Got image name: nextcloud/all-in-one:latest"
time="2024-07-29T19:25:59Z" level=debug msg="Checking if pull is needed" container=/nextcloud-aio-mastercontainer image="nextcloud/all-in-one:latest"
time="2024-07-29T19:25:59Z" level=debug msg="Built challenge URL" URL="https://index.docker.io/v2/"
time="2024-07-29T19:25:59Z" level=debug msg="Got response to challenge request" header="Bearer realm=\"https://auth.docker.io/token\",service=\"registry.docker.io\"" status="401 Unauthorized"
time="2024-07-29T19:25:59Z" level=debug msg="Checking challenge header content" realm="https://auth.docker.io/token" service=registry.docker.io
time="2024-07-29T19:25:59Z" level=debug msg="Setting scope for auth token" image=docker.io/nextcloud/all-in-one scope="repository:nextcloud/all-in-one:pull"
time="2024-07-29T19:25:59Z" level=debug msg="No credentials found."
time="2024-07-29T19:26:00Z" level=debug msg="Parsing image ref" host=index.docker.io image=nextcloud/all-in-one normalized=docker.io/nextcloud/all-in-one tag=latest
time="2024-07-29T19:26:00Z" level=debug msg="Doing a HEAD request to fetch a digest" url="https://index.docker.io/v2/nextcloud/all-in-one/manifests/latest"
time="2024-07-29T19:26:00Z" level=debug msg="Found a remote digest to compare with" remote="sha256:1789869aed9e05f0e5a65f3b4e14766977c2f9842b22f4ba4ac79205c10d6347"
time="2024-07-29T19:26:00Z" level=debug msg=Comparing local="sha256:d378d79e477bd2576f72671a15a6aef924be5ece6e256a9721d4b05aab24929d" remote="sha256:1789869aed9e05f0e5a65f3b4e14766977c2f9842b22f4ba4ac79205c10d6347"
time="2024-07-29T19:26:00Z" level=debug msg="Digests did not match, doing a pull."
time="2024-07-29T19:26:00Z" level=debug msg="Pulling image" container=/nextcloud-aio-mastercontainer image="nextcloud/all-in-one:latest"
time="2024-07-29T19:26:01Z" level=info msg="Found new nextcloud/all-in-one:latest image (6ad3a2878283)"
time="2024-07-29T19:26:01Z" level=info msg="Stopping /nextcloud-aio-mastercontainer (a31290d0df81) with SIGTERM"
time="2024-07-29T19:26:01Z" level=error msg="Error response from daemon: Cannot kill container: a31290d0df8145230eca1f970451b53efdfec187af650b199477ecba58f63b3e: Cannot kill container a31290d0df8145230eca1f970451b53efdfec187af650b199477ecba58f63b3e: unknown error after kill: runc did not terminate successfully: exit status 1: unable to signal init: permission denied\n: unknown"
time="2024-07-29T19:26:01Z" level=info msg="Session done" Failed=1 Scanned=1 Updated=0 notify=no
time="2024-07-29T19:26:01Z" level=info msg="Waiting for the notification goroutine to finish" notify=no

**Comment by szaimen** [comment, marked as answer]
> unknown error after kill: runc did not terminate successfully: exit status 1: unable to signal init: permission denied

Hi, I found this: https://www.google.com/search?q=unknown%20error%20after%20kill%3A%20runc%20did%20not%20terminate%20successfully%3A%20exit%20status%201%3A%20unable%20to%20signal%20init%3A%20permission%20denied

**Reply by MK796** [reply]
Hello @szaimen ,

That solved it! Thank you very much!
